scripts/main.bro

```diff
@@ -79,6 +83,8 @@ function effective_tld(domain: string): string
 function effective_domain(domain: string): string
 	{
 	local depth=2;
+	if ( effective_tlds_4th_level in domain )
+		depth=5;
 	if ( effective_tlds_3rd_level in domain )
```

Comment:
Could you also update the domain-tld.bro script?

Reply:
Looks like we were both suggesting the change at 88, but I take it mine did not show up on your end. I think there was maybe one more step in committing the suggestion. 

The tld-data.bro script?